### PR TITLE
docs: add customer.io brand header to README

### PR DESCRIPTION
**Pull request by joepurdy**
This PR updates the README for our Python client library to include our brand logo and mission similar to how we have on language specific integration libraries like https://github.com/customerio/customerio-ruby#readme

Let me know if there's any questions or suggestions!

### CODE REVIEW

**Review by idawes — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+  <p align="center">Power automated communication that people like to receive.</p>
+</p>
+
+![PyPI](https://img.shields.io/pypi/v/customerio)
+![Software License](https://img.shields.io/github/license/customerio/customerio-python)
+[![Build status](https://github.com/customerio/customerio-python/actions/workflows/main.yml/badge.svg)](https://github.com/customerio/customerio-python/actions/workflows/main.yml)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/customerio)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/customerio)
+
+# Customer.io Python
```
Comment (idawes) [inline README.md:14]: Did you intentionally drop the word "bindings" here?

Comment (joepurdy) [reply README.md:14]: Yeah, it was an outlier in that none of the other libraries identify themselves as "[language] bindings". So I've opted to standardize on everything being "Customer.io [language]" and avoided calling them anything more like "Customer.io [language] client" or "Customer.io [language] bindings".

I haven't merged the similar PRs to the [Node](https://github.com/customerio/customerio-node/pull/90), and [Go](https://github.com/customerio/go-customerio/pull/31) libraries yet (and won't until Monday at the earliest) so if there's a strong desire to call these libraries bindings that's fine by me. I care less about the syntax and more about consistency.

**Comment by idawes** [comment]
I’m certainly not tied to the “bindings” language. Consistency is good.

Cheers,

 Ian

**Pull request merged**